sample/common/build.gradle

```diff
@@ -66,16 +67,16 @@ kotlin {
   }
 }
 
-task iosTest {
-  def device = project.findProperty("iosDevice")?.toString() ?: "iPhone 8"
-  dependsOn 'linkDebugTestIos'
-  group = JavaBasePlugin.VERIFICATION_GROUP
-  description = "Runs tests for target 'ios' on an iOS simulator"
-
-  doLast {
-    def binary = kotlin.targets.ios.binaries.getTest('DEBUG').outputFile
-    exec {
-      commandLine 'xcrun', 'simctl', 'spawn',  '--standalone', device, binary.absolutePath
-    }
-  }
-}
+//task iosTest {
```

Comment:
I didn't know what to do here? Is this create automatically now so we should just delete it?

Reply:
it should be there automatically now, i think we can just remove this block

Reply:
Yeah, removed